etc/kayobe/stackhpc.yml

```
@@ -27,6 +27,10 @@ stackhpc_include_os_minor_version_in_repo_url: false
 
 stackhpc_rocky_9_url_version: "{{ '9.' + stackhpc_pulp_repo_rocky_9_minor_version | string if stackhpc_include_os_minor_version_in_repo_url | bool else '9' }}"
 
+# Pick build architecture for kolla and disk image builder
+# Choose between x86 or arm64
+stackhpc_cpu_arch: 'x86'
```

```suggestion
stackhpc_cpu_arch: 'x86_64'
```

https://docs.openstack.org/kayobe/2024.2/configuration/reference/kolla.html#global-configuration